src/mtl/include/mtl/MTLFormula.hpp

```diff
@@ -34,6 +35,17 @@ std::ostream &
 operator<<(std::ostream &out, const logic::MTLFormula<APType> &f)
 {
 	using logic::LOP;
+	using utilities::arithmetic::BoundType;
+	const auto print_until =
+	  [](std::ostream &out, const logic::MTLFormula<APType> &f, std::string_view op_symbol) {
+		  out << "(" << f.get_operands().front() << " " << op_symbol;
+		  const auto &interval = f.get_interval();
+		  if (interval.lowerBoundType() != BoundType::INFTY
+		      || interval.upperBoundType() != BoundType::INFTY) {
```

Comment:
What about an outstream-operator for intervals? I will add one.

Reply:
We already have one: ab1545023e7ca4a4bdafa841574445a3abd45c88
However, I want to omit the interval bounds if the interval is (INFTY, INFTY), that's why the check is here.

Reply:
See also bf6df38f1c6c4e8ff35e126379131c2c4558bfb8

Reply:
I added a comment that should make this clear :)